### PR TITLE
[ROCm] Create a wrapper header for rocprim and cub

### PR DESCRIPTION
**Pull request by ekuznetsov139**
This PR creates a header that includes either rocprim or cub, depending on the selected build architecture.

### CODE REVIEW

**Comment by ekuznetsov139** [comment]
@nvining-work

**Comment by deven-amd** [comment]
gentle ping

**Comment by gbaned** [comment]
@ekuznetsov139 Can you please resolve conflicts? Thanks!

**Comment by ekuznetsov139** [comment]
Done

**Comment by deven-amd** [comment]
gentle ping

File: tensorflow/core/kernels/BUILD [hunk 1]
```diff
@@ -517,6 +517,16 @@ cc_library(
     deps = ["//third_party/eigen3"],
 )
 
+cc_library(
```
Comment (cheshire) [inline tensorflow/core/kernels/BUILD]: Would `cub_or_rocprim` be a better name?

**Review by cheshire — APPROVED** [review]

File: tensorflow/core/kernels/BUILD (hunk 1 above)
Comment (ekuznetsov139) [reply tensorflow/core/kernels/BUILD]: I think `gpu_prim_hdrs` better reflects the purpose of the package. However, I can rename it as long as it does not take another 1.5 months to review the PR after the change.

Comment (cheshire) [reply tensorflow/core/kernels/BUILD]: Point taken, but I wasn't really able to work for the last couple of weeks, and I think the PR got stuck in limbo.

**Review by cheshire — DISMISSED** [review]

File: tensorflow/core/kernels/BUILD (hunk 1 above)
Comment (ekuznetsov139) [reply tensorflow/core/kernels/BUILD]: It's not you personally, it's the entire system. I still have open PRs originally submitted in early January. Any change (however minor) seems to kick the PR back to the end of the queue and then it takes up to a month for it to be reviewed again. So I am wary of making any changes.

However, in this case, I had to fix buildifier formatting anyway, I can do the renaming as well if you like.

Comment (cheshire) [reply tensorflow/core/kernels/BUILD]: > Any change (however minor) seems to kick the PR back to the end of the queue
I'm not sure that's how it works, usually once the reviewer is looking into it, the turnaround should be quite reasonable.

The system (kinda sorta) works when a correct reviewer is assigned (either manually, or by the triaging team). At this point, for AMD-specific changes it's probably either @chsigg or me (though I normally only review XLA-specific stuff, but I can also help with general reviews).

**Review by cheshire — APPROVED** [review]

**Pull request merged**